osu.Game/Users/UserStatistics.cs

```diff
@@ -118,5 +125,38 @@ public int this[ScoreRank rank]
                 }
             }
         }
+        public enum GameVariant
+        {
+            [EnumMember(Value = "4k")]
+            FourKey,
+            [EnumMember(Value = "7k")]
+            SevenKey
+        }
+
+        public class Variant
+        {
+            [JsonProperty("country_rank")]
+            public int? CountryRank;
+
+            [JsonProperty("global_rank")]
+            public int? GlobalRank;
+
+            [JsonProperty("mode")]
+            public string Mode;
+
+            [JsonProperty("pp")]
+            public decimal PP;
+
+            [JsonProperty("variant")]
+            [JsonConverter(typeof(StringEnumConverter))]
+            public GameVariant? VariantType;
+
+            public LocalisableString VariantDisplay => VariantType switch
+            {
+                GameVariant.FourKey => CommonStrings.FourKey,
+                GameVariant.SevenKey => CommonStrings.SevenKey,
+                _ => string.Empty
+            };
```

Comment:
This can just be a `LocalisableDescription` attribute in the enum members, with `GetLocalisableDescription()`.